examples/Demo/Shared/Pages/Wizard/Examples/WizardEditForms.razor

```diff
@@ -0,0 +1,123 @@
+﻿@using System.ComponentModel.DataAnnotations
+
+@inherits Microsoft.AspNetCore.Components.ComponentBase
+
+@inject IDialogService DialogService
+
+<FluentWizard StepperPosition="StepperPosition.Left"
+              DisplayStepNumber="@(WizardStepStatus.Current | WizardStepStatus.Next)"
+              Border="WizardBorder.Outside"
+              StepTitleHiddenWhen="@GridItemHidden.XsAndDown"
+              Height="auto"
+              Style="min-height: 300px;"
+              OnFinish="@OnFinishedAsync">
+    <Steps>
+        <FluentWizardStep Label="Personal Info"
+                          OnChange="@OnStepChange">
+            <EditForm Model="_formData1" FormName="personalInfo" method="post">
+                <DataAnnotationsValidator />
+                <FluentWizardStepFormValidator />
```

Comment:
What is the goal of the `FluentWizardStepFormValidator`?
Apart from registering the EditContext with the WizardStep, which is more a "technical" reason.

In other words, what will be the result if I use `EditForm` without `FluentWizardStepFormValidator`? The `DataAnnotationsValidator` will not be enough?

Is it the single possibility?

Reply:
The `FluentWizardStepFormValidator` is what enables the `WizardPage` to become aware of the edit context so that it can call the `EditContext.Validate()` method before moving to the next page.  If you leave this out then the default behavior is maintained.  If you are aware of a better way to inform the `WizardPage` that it contains an `EditForm` with an `EditContext` then let me know.

Reply:
I don't know 😊
I'd like to know if this is the only solution.

It would be easier not to have to add `DataAnnotationsValidator` and also `FluentWizardStepFormValidator`. But the question is indeed how to retrieve EditContext 😇

I can check this weekend

Reply:
I'd like to avoid adding something too quickly, without knowing if it's possible to make it simpler.

Reply:
Agreed.  The only other way I know of is to instantiate your own `EditContext` higher up the chain and then pass it down to the different components via a parameter (either explicit or cascading).  But I'll let you come back after the weekend to confirm if that's the case.  Also, the `<DataAnnotationsValidator>` is not required per se by this change, but is rather the default/best practice for validating a Blazor `EditForm` when using data annotations.  For more info see here: [https://learn.microsoft.com/en-us/aspnet/core/blazor/forms/?view=aspnetcore-8.0](https://learn.microsoft.com/en-us/aspnet/core/blazor/forms/?view=aspnetcore-8.0)

I personally have created my own higher-level component called `<FluentForm>` which encapsulates an `EditForm` and includes a lot of the defaults for editing forms and I include the `<DataAnnotationsValidator>` as well.  I would also place the `FluentWizardStepFormValidator` in it so that I don't need to add it everywhere I use an `EditForm`.

Reply:
It might be another (and good) idea to add a `FluentEditForm`. It might simplify this and other (future or existing) features. @vnbaaij  what is your feeling about EditForm?

Give me a few days to do some tests. Tomorrow is my son's birthday, but I'll try to find some time on Sunday :-)